Registry/Registry.EM_COMMON

```diff
@@ -1072,6 +1072,44 @@ state   real   OL2              ij     misc        1         -     i012rhdus "OL
+state   real   OA3ls            ij     misc        1         -     i012rhdus "OA3ls"                 "OROGRAPHIC ASYMMETRY in South-West direction from 2.5min-resolution topography"   ""
+state   real   OA4ls            ij     misc        1         -     i012rhdus "OA4ls"                 "OROGRAPHIC ASYMMETRY in North-West direction from 2.5min-resolution topography"   ""
+state   real   OL1ls            ij     misc        1         -     i012rhdus "OL1ls"                 "OROGRAPHIC EFFECTIVE LENGTH for Westerly flow from 2.5min-resolution topography"   ""
+state   real   OL2ls            ij     misc        1         -     i012rhdus "OL2ls"                 "OROGRAPHIC EFFECTIVE LENGTH for Southerly flow from 2.5min-resolution topography"   ""
+state   real   OL3ls            ij     misc        1         -     i012rhdus "OL3ls"                 "OROGRAPHIC EFFECTIVE LENGTH for South-Westerly flow from 2.5min-resolution topography"   ""
+state   real   OL4ls            ij     misc        1         -     i012rhdus "OL4ls"                 "OROGRAPHIC EFFECTIVE LENGTH for North-Westerly flow from 2.5min-resolution topography"   ""
+state   real   VAR2Dss          ij     misc        1         -     i012rhdus "VARss"                 "STANDARD DEVIATION of 30sec-resolution topography"   "m"
+state   real   OC12Dss          ij     misc        1         -     i012rhdus "CONss"                 "OROGRAPHIC CONVEXITY of 30sec-resolution topography"   ""
+state   real   OA1ss            ij     misc        1         -     i012rhdus "OA1ss"                 "OROGRAPHIC ASYMMETRY in West direction from 30sec-resolution topography"   ""
+state   real   OA2ss            ij     misc        1         -     i012rhdus "OA2ss"                 "OROGRAPHIC ASYMMETRY in South direction from 30sec-resolution topography"   ""
+state   real   OA3ss            ij     misc        1         -     i012rhdus "OA3ss"                 "OROGRAPHIC ASYMMETRY in South-West direction from 30sec-resolution topography"   ""
+state   real   OA4ss            ij     misc        1         -     i012rhdus "OA4ss"                 "OROGRAPHIC ASYMMETRY in North-West direction from 30sec-resolution topography"   ""
+state   real   OL1ss            ij     misc        1         -     i012rhdus "OL1ss"                 "OROGRAPHIC EFFECTIVE LENGTH for Westerly flow from 30sec-resolution topography"   ""
+state   real   OL2ss            ij     misc        1         -     i012rhdus "OL2ss"                 "OROGRAPHIC EFFECTIVE LENGTH for Southerly flow from 30sec-resolution topography"   ""
+state   real   OL3ss            ij     misc        1         -     i012rhdus "OL3ss"                 "OROGRAPHIC EFFECTIVE LENGTH for South-Westerly flow from 30sec-resolution topography"   ""
+state   real   OL4ss            ij     misc        1         -     i012rhdus "OL4ss"                 "OROGRAPHIC EFFECTIVE LENGTH for North-Westerly flow from 30sec-resolution topography"   ""
+
```

Comment:
@mdtoy 
Where do these input fields come from? They are not part of the standard WPS geogrid output.

Reply:
@mdtoy @weiwangncar @dudhia
If the new options are activated without any valid input data, is there any way for a user to know that they are just adding up a lot of zeros?

Reply:
Michael Duda is in the process of updating WPS to have these fields generated in geogrid.

For the possibility of a user running the drag suite without the required fields in wrfinput_d0x, currently there is no way for the user to know.  If they're not present in wrfinput_d0x, do the fields default to zero?  Is there a standard error that could be called if all the inputs are zero?  Of course, if one's domain doesn't include any land points, then these fields would be zero anyways.  Hopefully, the user wouldn't select the GWD scheme over the ocean.
Anyway, if you have any recommendations/preferences on how to handle this, I'd be happy to code it up.